x/group/client/testutil/tx.go

```diff
@@ -34,6 +34,7 @@ type IntegrationTestSuite struct {
 	groupPolicies []*group.GroupPolicyInfo
 	proposal      *group.Proposal
 	vote          *group.Vote
+	votedMember   *group.Member
```

Comment:
what about this instead? 
```suggestion
	voter   *group.Member
```